osconfig_tests/test_suites/guestpolicies/guest_policies_utils.go

```diff
@@ -87,6 +87,22 @@ while(1) {
 		ss = fmt.Sprintf(ss, utils.InstallOSConfigGooGet(), packageName, packageInstalled, packageNotInstalled)
 		key = "windows-startup-script-ps1"
 
+	case "zypper":
+		ss = `%s
+while true; do
+  isinstalled=$(/usr/bin/rpmquery -a %s)
+  if [[ $isinstalled =~ ^%s-* ]]; then
+	uri=http://metadata.google.internal/computeMetadata/v1/instance/guest-attributes/%s
+	curl -X PUT --data "1" $uri -H "Metadata-Flavor: Google"
+  else
+	uri=http://metadata.google.internal/computeMetadata/v1/instance/guest-attributes/%s
+	curl -X PUT --data "1" $uri -H "Metadata-Flavor: Google"
```

Comment:
It looks like the if and else blocks are identical. Should this one set data to "0"?